docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -394,6 +394,8 @@ const usdPrice: GridColTypeDef = {
 
 {{"demo": "pages/components/data-grid/columns/CustomColumnTypesGrid.js", "bg": "inline"}}
 
+> ⚠ If an unsupported column type is used the `__default__` column type will be used instead.
```

Comment:
Should we be more explicit here? I didn't know about this `__default__` column type.

```suggestion
> ⚠ If an unsupported column type is used the `string` column type will be used instead.
```